xen/arch/x86/include/asm/intel_txt.h

```diff
@@ -341,22 +325,51 @@ static inline int is_in_pmr(struct txt_os_sinit_data *os_sinit, uint64_t base,
 	return 0;
 }
 
+static inline void find_evt_log(void **evt_log, uint32_t *evt_log_size)
```

Comment:
Please add a comment that `evt_log` is physical address and it is up to the caller to map it to virtual, if needed.

Reply:
https://github.com/TrenchBoot/xen/compare/5ed262bc0df7696f824b5a547339a804fa9e5a52..39a1478db53029b133c75d4f15a0f5774ad92f8a#diff-dd4611d055ace3b3c87ffe90aaf480b0d4a0e4098adb6de5026e3144a4821b09R328-R329